bin/oref0-radio-tune.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+source $(dirname $0)/oref0-bash-common-functions.sh || (echo "ERROR: Failed to run oref0-bash-common-functions.sh. Is oref0 correctly installed?"; exit 1)
+
+usage "$@" <<EOF
+Usage: $self
+Determine the optimum frequency to communicate with the pump and write
+the output to monitor/mmtune.json and monitor/medtronic_frequency.ini
+EOF
+
+export MEDTRONIC_PUMP_ID=`grep serial pump.ini | tr -cd 0-9`
+OREF0_DEBUG=${OREF0_DEBUG:-0}
+if [[ "$OREF0_DEBUG" -ge 1 ]] ; then
+  exec 3>&1
+else
+  exec 3>/dev/null
+fi
+if [[ "$OREF0_DEBUG" -ge 2 ]] ; then
+  exec 4>&1
+  set -x
+else
+  exec 4>/dev/null
+fi
+
+function mmtune_Go() {
+  set -o pipefail
+  if ( grep "WW" pump.ini ); then
```

Comment:
this expected to be in `~/myopenaps` directory. I accidently did not and then you get this error:
```
# oref0-radio-tune
grep: pump.ini: No such file or directory
/usr/bin/oref0-radio-tune: line 34: monitor/mmtune.json: No such file or directory
mmtune: grep: pump.ini: No such file or directory
tee: monitor/mmtune.json: No such file or directory
2018/08/20 00:31:22 connected to CC111x radio on /dev/spidev5.1
2018/08/20 00:31:22 MEDTRONIC_PUMP_ID environment variable is not set
jq: error: Could not open file monitor/mmtune.json: No such file or directory
```

Either add `cd ~/myopenaps ($directory)` in the script or give a more userfriendly error message like. Should be started in your oref0 directory `$HOME/myopenaps`